database/youtube/web-development.json

```diff
@@ -125,5 +125,13 @@
         "category": "youtube",
         "subcategory": "web-development",
         "language": "english"
+    },
+    {
+        "name": "Coder Coder",
```

Comment:
You can change the name to `TheCoderCoder` it would be more relevant

Reply:
ok